docs/admin-guide.md

```diff
@@ -0,0 +1,42 @@
+# ChainPatrol
+
+### About this app
+
+The [ChainPatrol](https://chainpatrol.io/) mini-app gives your community members
+access to security tools to check URLs for scams and report malicious activity
+on your Discord server. Use the `/chainpatrol <sub-command>` commands to empower
+your community to protect themselves from phishing attacks and other scams.
+
+### Getting Started
+
+To get started, install the ChainPatrol mini-app from the
+[Collab.Land Marketplace](https://cc.collab.land/).
+
+Your community members can then start using `/chainpatrol check` to start
+checking URLs whenever they are unsure of a message that has been sent in your
```

Comment:
> Your community members can then start using `/chainpatrol check` to start
checking URLs whenever they are unsure of a message that has been sent in your
Discord server

Your community members can then use `/chainpatrol check` to check suspect URLs against the <server/global?> scam list when a suspicious message is sent in your Discord server.

Reply:
Report scam URLs using the `/chainpatrol report` command to report the scam to ChainPatrol.